improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -655,78 +664,80 @@ def __init__(self, distribution="norm", shape_parameters=None):
 
-          b\\_rescaled = (b - mean)/standard\\_deviation
+          b\\_rescaled = (b - location\\_parameter)/scale\\_parameter
 
         Please see :data:`scipy.stats.truncnorm` for some further information.
 
         Args:
-            mean (numpy.ndarray):
-                Mean to be used to scale the shape parameters.
-            std (numpy.ndarray):
-                Standard deviation to be used to scale the shape parameters.
+            location_parameter (numpy.ndarray):
+                Location parameter to be used to scale the shape parameters.
+            scale_parameter (numpy.ndarray):
+                Scale parameter to be used to scale the shape parameters.
 
         """
         if self.distribution.name == "truncnorm":
             if self.shape_parameters:
                 rescaled_values = []
                 for value in self.shape_parameters:
-                    rescaled_values.append((value - mean) / std)
+                    rescaled_values.append((value - location_parameter) /
+                                           scale_parameter)
                 self.shape_parameters = rescaled_values
             else:
                 msg = ("For the truncated normal distribution, "
                        "shape parameters must be specified.")
                 raise ValueError(msg)
 
 
-class GeneratePercentilesFromMeanAndVariance(BasePlugin, FromMeanAndVariance):
+class ConvertLocationAndScaleParametersToPercentiles(
+        BasePlugin, ConvertLocationAndScaleParameters):
     """
-    Plugin focussing on generating percentiles from mean and variance.
-    In combination with the EnsembleReordering plugin, this is Ensemble
-    Copula Coupling.
+    Plugin focusing on generating percentiles from location and scale
+    parameters. In combination with the EnsembleReordering plugin, this is
+    Ensemble Copula Coupling.
     """
 
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
-        result = ('<GeneratePercentilesFromMeanAndVariance: '
+        result = ('<ConvertLocationAndScaleParametersToPercentiles: '
                   'distribution: {}; shape_parameters: {}>')
         return result.format(self.distribution.name, self.shape_parameters)
 
-    def _mean_and_variance_to_percentiles(
-            self, calibrated_forecast_predictor, calibrated_forecast_variance,
-            template_cube, percentiles):
+    def _location_and_scale_parameters_to_percentiles(
+            self, location_parameter, scale_parameter, template_cube,
+            percentiles):
         """
-        Function returning percentiles based on the supplied
-        mean and variance. The percentiles are created by assuming a
+        Function returning percentiles based on the supplied location and
+        scale parameters. The percentiles are created by assuming a
         Gaussian distribution and calculating the value of the phenomenon at
         specific points within the distribution.
```

Comment:
As an aside, does the assumption of a gaussian distribution here make this inappropriate for precip/wind etc.? I think we chatted about this before, but I don't recall the outcome of the conversation.

Reply:
Apologies, this sentence is actually out-of-date following #1009, which added the ability to pass in any distribution supported by scipy when recreating percentiles or probabilities from the mean and variance. I've removed this sentence.